python/cudf/comm/serialize.py

```diff
@@ -1,8 +1,8 @@
-# A flag to allow dask_gdf to detect and warn if
-# IPC serialization is unavailable
-CUSTOM_SERIALIZATION_AVAILABLE = False
-
+import functools
 
 def register_distributed_serializer(cls):
```

Comment:
It looks like we need to do a bit more work to make this a zero op if `distributed` isn't available in the user's environment.